src/integration-test/tests/citizen/claimantResponse/pages/claimant-settle-the-claim-defendant-admitted.ts

```diff
@@ -0,0 +1,25 @@
+import I = CodeceptJS.I
+
+const I: I = actor()
+
+const fields = {
+  yes: 'input[id=admittedyes]',
```

Comment:
use optionYes , see claimant-interest.ts 
or inside fields constant define the radiobutton constant like:
```
radioButtons : {
     optionYes : 'input[id=admittedyes]',
     optionNo : ''input[id=admittedyes]'
}
```

Reply:
or rename radioButtons to settleClaim

Reply:
Done